odoo/cli/server.py

```diff
@@ -66,6 +66,11 @@ def report_configuration():
     port = config['db_port'] or os.environ.get('PGPORT', 'default')
     user = config['db_user'] or os.environ.get('PGUSER', 'default')
     _logger.info('database: %s@%s:%s', user, host, port)
+    if sys.version_info[:2] > odoo.MAX_PY_VERSION:
```

Comment:
We should handle MIN and MAX version the same way. Either in the assert (error during import) or both as a warning (and not only in the `cli`), but at the end of importing the `odoo` module. Unless there is a reason to do it differently.

Reply:
I don't agree, while it is *guaranteed* it won't work (e.g. for a syntax error) with previous versions, there is actually a chance that it works with newer ones

Reply:
Some parts may indeed continue to work. Some parts of the `ast` module may not work.